<map id="pip._vendor.requests.exceptions.ConnectionError" name="pip._vendor.requests.exceptions.ConnectionError">
<area shape="rect" id="node4" href="$classpip_1_1__vendor_1_1requests_1_1exceptions_1_1ConnectTimeout.html" title="pip._vendor.requests.exceptions.\lConnectTimeout" alt="" coords="643,5,857,47"/>
<area shape="rect" id="node5" href="$classpip_1_1__vendor_1_1requests_1_1exceptions_1_1ProxyError.html" title="pip._vendor.requests.exceptions.\lProxyError" alt="" coords="643,71,857,112"/>
<area shape="rect" id="node6" href="$classpip_1_1__vendor_1_1requests_1_1exceptions_1_1SSLError.html" title="pip._vendor.requests.exceptions.\lSSLError" alt="" coords="643,136,857,177"/>
<area shape="rect" id="node2" href="$classpip_1_1__vendor_1_1requests_1_1exceptions_1_1RequestException.html" title="pip._vendor.requests.exceptions.\lRequestException" alt="" coords="117,71,332,112"/>
</map>
